src/components/VmuiLink.tsx

```diff
@@ -66,27 +66,24 @@ const VmuiLink: FC<Props> = ({
 
   useEffect(() => {
     const getExternalLink = async () => {
-      if (!panelData?.request) {
-        return
-      }
-
       const dataSourceSrv = getBackendSrv();
       const dsSettings = await dataSourceSrv.get(`/api/datasources/${datasource.id}`);
       let relativeTimeId = 'none'
 
-      const {
-        request: { range, interval, scopedVars, rangeRaw },
-      } = panelData;
+      const timeRange = panelData?.timeRange || getDefaultTimeRange();
```

Comment:
is this panelData present on version >10?

Reply:
`panelData` is present in versions >10, but it may be absent in some cases, such as when creating alerts.

Reply:
ok, lets use it as is right now